src/main/resources/application.properties

```diff
@@ -9,5 +9,10 @@ spring.jpa.hibernate.ddl-auto=update
 # Enabling H2 Console localhost:8080/h2-console
 spring.h2.console.enabled=true
 # debug=true
-
 server.require-ssl=false
+
+spring.jpa.show-sql=true
+logging.level.org.hibernate.sql=debug
```

Comment:
tady ty log levely taky prosimte revertuj

Reply:
Hmm, nedaval bych do gitu prosim. Aspon pokud to nebyla domluva.